Simplify GMT_RGBchart.sh by only producing tabloid size in modern mode

meghanrjones opened the pull request:
**Description of proposed changes**

This PR simplifies the script for GMT colors to only produce a single size using modern mode (tabloid). Although there is  now only a single button to download the tabloid output, users can download/modify the modern mode script to produce a figure of any size.

Prerequisite of https://github.com/GenericMappingTools/gmt/issues/6356.

<!-- Please describe changes proposed and **why** you made them. If unsure, open an issue first so we can discuss.-->

<!-- If fixing an issue, put the issue number after the # below (no spaces). Github will automatically close it when this gets merged. -->
Fixes #


**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

seisman commented:
FYI, test `doc/scripts/GMT_RGBchart.sh` fails on Linux/macOS/Windows, likely due to the changes in this PR.

meghanrjones commented:
> FYI, test `doc/scripts/GMT_RGBchart.sh` fails on Linux/macOS/Windows, likely due to the changes in this PR.

The issue is that https://github.com/GenericMappingTools/gmt/pull/6548 reverted the changes to the RGB files made in this PR as shown by the dvc diff output below. We should come up with a way to prevent this situation, but to fix the problem:
1. Remove GMT_RGBchart_a4, GMT_RGBchart_letter, and GMT_RGBchart_tabloid from doc/scipts/images.
2. Copy the passing output from the GMT_RGBchart test to doc/scripts/images
3. dvc add doc/scripts/images
4. git commit doc/scripts/images.dvc
5. dvc push
6. git push to a PR

```
dvc diff 31d7f9544fd0ffb60aaa3d359a99a96a992df647 129849b4bcf7614313313c550492572089b7157e                                                                                          
Added:                                                                                                        
    doc/scripts/images/GMT_RGBchart_a4.ps                                                                     
    doc/scripts/images/GMT_RGBchart_letter.ps                                                                 
    doc/scripts/images/GMT_RGBchart_tabloid.ps                                                                
    doc/scripts/images/GMT_seamount_azim.ps                                                                   
    doc/scripts/images/GMT_seamount_psi.ps                                                                    
    doc/scripts/images/GMT_seamount_slide.ps                                                                  
    doc/scripts/images/GMT_seamount_specs.ps                                                                  
    doc/scripts/images/GMT_seamount_u0.ps                                                                     
                                                                                                              
Modified:                                                                                                     
    doc/scripts/images/                                                                                       
    doc/scripts/images/GMT_RGBchart.ps
    doc/scripts/images/GMT_seamount_cum_inc.ps
    doc/scripts/images/GMT_seamount_flux.ps
    doc/scripts/images/GMT_seamount_prisms.ps
    doc/scripts/images/GMT_seamount_types.ps
    test/baseline/potential/
    test/baseline/potential/admittance.ps
    test/baseline/potential/firmoviscous.ps
    test/baseline/potential/firmoviscous2.ps
    test/baseline/potential/flex_inplane.ps
    test/baseline/potential/flexure_e.ps
    test/baseline/potential/flexure_en.ps
    test/baseline/potential/flexure_gl.ps
    test/baseline/potential/flexure_ve.ps
    test/baseline/potential/growth_g.ps
    test/baseline/potential/gseamounts.ps
```
